include/picongpu/plugins/adios/ADIOSWriter.hpp

```diff
@@ -111,11 +111,17 @@ int64_t defineAdiosVar(int64_t group_id,
 {
     int64_t var_id = 0;
 
+    std::string revertedDimensions const =
```

Comment:
ah nooo... just outermost right of the type
```
std::string const revertedDimensions = ...
```

Reply:
ohh ahmm .. let's agree this didn't happen